### PR TITLE
Flux Index Bugs

### PR DESCRIPTION
**Pull request by ajnonaka**
Thanks to Changho for spotting both of these.

Fixed a bug in the z ChiX stencil.

Fixed an error in the z heat flux indexing which doesn't ultimately affect the simulation since it's summed into the same location in memory in the end anyway regression tests will likely change when this is merged.

### CODE REVIEW

**Comment by ajnonaka** [comment]
What line 1364 modification are you suggesting? @isriva

**Comment by isriva** [comment]
@ajnonaka line 1364 of flucStag.cpp should be:
`Real ChiX = 0.5*(chi(i,j,k-1,ns)*prim(i,j,k-1,6+nspecies+ns)+chi(i,j,k,ns)*prim(i,j,k,6+nspecies+ns));`

File: src_compressible_stag/fluxStag.cpp [hunk 1]
```diff
@@ -1361,7 +1361,7 @@ void calculateFluxStag(const MultiFab& cons_in, const std::array< MultiFab, AMRE
                             meanYk[ns] = 0.5*(prim(i,j,k-1,6+ns)+prim(i,j,k,6+ns));
                             Real term2 = (meanXk[ns]-meanYk[ns])*(prim(i,j,k,5)-prim(i,j,k-1,5))/dx[2]/meanP;
                             dk[ns] = term1 + term2;
-                            Real ChiX = 0.5*(chi(i,j,k,ns)*prim(i,j,k-1,6+nspecies+ns)+chi(i,j,k+1,ns)*prim(i,j,k,6+nspecies+ns));
+                            Real ChiX = 0.5*(chi(i,j,k,ns)*prim(i,j,k-1,6+nspecies+ns)+chi(i,j,k,ns)*prim(i,j,k,6+nspecies+ns));
```
Comment (ajnonaka) [inline src_compressible_stag/fluxStag.cpp:1364]:
```suggestion
                            Real ChiX = 0.5*(chi(i,j,k-1,ns)*prim(i,j,k-1,6+nspecies+ns)+chi(i,j,k,ns)*prim(i,j,k,6+nspecies+ns));
```

**Comment by ajnonaka** [comment]
@isriva fixed!

**Pull request merged**